cabot_alert_hipchat/tests/test_hipchat.py

```diff
@@ -36,7 +34,7 @@ def test_normal_alert(self, fake_hipchat_alert):
         self.service.old_overall_status = Service.ERROR_STATUS
         self.service.save()
         self.service.alert()
-        fake_hipchat_alert.assert_called_with(u'Service Service is back to normal: http://localhost/service/1/. @test_user_hipchat_alias', color='green', sender='Cabot/Service')
+        fake_hipchat_alert.assert_called_with(u'Service Service is back to normal: http://localhost/service/1/.  @test_user_hipchat_alias', color='green', sender='Cabot/Service')
```

Comment:
To get the tests to pass :)

I'm not sure when it changed but it didn't seem worth investigating